cosmo_tester/framework/cfy_helper.py

```diff
@@ -97,6 +101,34 @@ def bootstrap(self,
                 task_retry_interval=task_retry_interval,
                 verbose=verbose,
                 debug=debug).wait()
+            self.upload_plugins(cfy)
+
+    def _get_wagons(self, wagons=None):
+        self.logger.info('Downloading Wagons...')
+
+        wagon_paths = []
+        versions_repo_url = 'https://github.com/geokala/' \
+                            'cloudify-versions.git'
+        path = git_helper.clone(
+            versions_repo_url, self._cfy_workdir,
+            branch=os.environ.get('CLOUDIFY_VERSIONS_BRANCH',
+                                  'master'))
+        url_path = os.path.join(path, 'plugin-urls.yaml')
+        plugins = get_yaml_as_dict(url_path)['plugins']
+        for plugin in plugins:
+            if (plugin['name'] or plugin['wgn_url']) in wagons or not wagons:
```

Comment:
(might need to check with Nir as he wrote this bit, but having just looked in more depth at it):
We're not using the functionality to allow filtering what wagons to install yet, so we could probably remove this and implement it later when it is wanted.

Otherwise, it seems designed to accept a list of plugin names or URLs... but in that case I think actually providing a different location to get the plugin URLs from would be more sensible.

Yeah, I'll remove this for the moment.